functions/src/notifications/populateBillNotificationEvents.ts

```diff
@@ -12,17 +12,25 @@ import { BillHistory } from "../bills/types"
 // Get a reference to the Firestore database
 const db = admin.firestore()
 
-type Notification = {
+export type Notification = {
   type: string
-  court: string
-  id: string
-  name: string
-  history: BillHistory
-  historyUpdateTime: Timestamp
+
+  billCourt: string
+  billId: string
+  billName: string
+
+  billHistory: BillHistory
+
+  testimonyUser: string
+  testimonyPosition: string
+  testimonyContent: string
+  testimonyVersion: number
+  
+  updateTime: Timestamp
```

Comment:
Could we leave comments on which fields correspond to which type? i.e. billHistory is for bills, then the testimony fields are for testimony, and which ones are used for both types of Notifications.